docs/source/en/model_doc/albert.md

```diff
@@ -60,13 +60,80 @@ Next sentence prediction is replaced by a sentence ordering prediction: in the i
 This model was contributed by [lysandre](https://huggingface.co/lysandre). This model jax version was contributed by
 [kamalkraj](https://huggingface.co/kamalkraj). The original code can be found [here](https://github.com/google-research/ALBERT).
 
-## Documentation resources
+## Resources
 
-- [Text classification task guide](../tasks/sequence_classification)
-- [Token classification task guide](../tasks/token_classification)
-- [Question answering task guide](../tasks/question_answering)
+The resources provided in the following sections consist of a list of official Hugging Face and community (indicated by 🌎) resources to help you get started with AlBERT. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+
+<PipelineTag pipeline="text-classification"/>
+
+- A blog post on how to train AlBERT with Blurr for sequence classification can be done by adapting this [example](https://huggingface.co/blog/fastai).
+- A blog post on how to use [Ray to tune hyperparameters](https://huggingface.co/blog/ray-tune) can be adapted for AlBERT.
+- A notebook on how to [Fine-tune ALBERT for sentence-pair classification](https://github.com/NadirEM/nlp-notebooks/blob/master/Fine_tune_ALBERT_sentence_pair_classification.ipynb). 🌎
+- A notebook on how to [Fine Tuning Transformer for MultiLabel Text Classification](https://github.com/abhimishra91/transformers-tutorials/blob/master/transformers_multi_label_classification.ipynb)
+
+- Adapt this blog post on [Getting Started with Sentiment Analysis using Python](https://huggingface.co/blog/sentiment-analysis-python) for AlBERT.
+
+- [`AlBertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-classification).
```

Comment:
The "B" should be lowercase (this typo occurs several times so I would double check to make sure they're all lowercase)

```suggestion
- [`AlbertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-classification).
```

Reply:
Ok sure :)